es6-promises/index.js

```diff
@@ -0,0 +1,11 @@
+const takeAChance = require('./take-a-chance');
+
+const promiseObj = takeAChance('Monique');
+
+promiseObj.then(value => {
+  console.log(value);
+});
+
+promiseObj.catch(Error => {
+  console.log(Error.message);
+});
```

Comment:
Please avoid naming your parameters the same as built-in globals: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error